packages/perseus/src/widgets/grapher/grapher-validator.ts

```diff
@@ -7,10 +7,10 @@ import type {
 } from "../../validation.types";
 
 function grapherValidator(
-    state: PerseusGrapherUserInput,
+    userInput: PerseusGrapherUserInput,
```

Comment:
Was there a reason to keep userInput as state? 🤔 I vaguely remember reverting a change like this, but not can't remember the reasoning.

Reply:
“What's in a name? That which we call a rose by any other name would smell just as sweet.”

I prefer `userInput` over `state` personally